sdks/java/core/src/main/java/org/apache/beam/sdk/transforms/Combine.java

```diff
@@ -289,6 +305,20 @@ private Combine() {
    * arbitrary tree structure. Commutativity is required because any order of the input values is
    * ignored when breaking up input values into groups.
    *
+   * <h3>Note on Data Encoding</h3>
+   *
+   * <p>Some form of data encoding is required when using custom types in a CombineFn which do not
```

Comment:
It would be better to say something along the lines:

We are relying on the fact that there is a generic `CoderProvider` that is able to provide a coder for any `Serializable`. In general there are two ways in which a coder for the accumulator type is provided:
* It can be provided by the default CoderRegistry. See the CoderRegistry for the numerous ways in which to bind a type to a coder.
* The `getAccumulatorCoder` method is overridden and provides the coder.

Reply:
Included these additional comments.   PTAL? 

Thanks!